src/test/addresses_goerli.sol

```diff
@@ -42,7 +42,8 @@ contract Addresses {
         addr["MCD_DOG"]                         = 0x5cf85A37Dbd28A239698B4F9aA9a03D55C04F292;
         addr["MCD_VOW"]                         = 0x23f78612769b9013b3145E43896Fa1578cAa2c2a;
         addr["MCD_JOIN_DAI"]                    = 0x6a60b7070befb2bfc964F646efDF70388320f4E0;
-        addr["MCD_FLAP"]                        = 0x015bEd3a7EBbB0Be03A35E0572E8a7B0BA2AA0fB;
```

Comment:
It is a replacement, you can't keep 2 flaps through the `vow`.
All what is needed should be done. What do you think is missing?

Reply:
Regarding de-auth: I can't see where VAT is being `deny`ed in the spell, but I see `can` is being checked in the test, so it should be fine.

I understand that you can't have 2 flappers operating simultaneously. But according to the archive old core contracts [are sometimes added](https://github.com/makerdao/spells-goerli/blob/27dc2da58640432d924dca586d7658458ab3188a/src/test/addresses_goerli.sol#L75) to the chainlog with `_LEGACY` suffix for administrative reasons. The question is if it should happen with the old flapper as well?

Reply:
None the new or the old Flapper are relied in the `vat`. There is a `hope` being called from the `vow` to the `flapper` and that happens in its `file` function https://github.com/makerdao/dss/blob/v1.2/src/vow.sol#L104

Reply:
Just confirmed with GovAlpha, but leaving it here as well. We won't keep the old Flapper as legacy because it won't be operative anymore. The legacy thing has been used to keep contracts that are older version but still working in a certain extent.